lib/core/alice_core.dart

```diff
@@ -1,7 +1,7 @@
 import 'dart:async';
 
-import 'package:alice_lightweight/model/alice_http_error.dart';
 import 'package:alice_lightweight/model/alice_http_call.dart';
+import 'package:alice_lightweight/model/alice_http_error.dart';
```

Comment:
I suggest to not:

1. Organize imports
2. Re-format code

This is so that should you decide to rebase from the original repo, it's a lot easier.

Reply:
Can run `flutter format .` revert the re-format code because that would be the default line length (80).

Reply:
Done remove unwanted changes caused by IDE and formatter~